This notebook verifies if the various simulated quantities are realistic for observations. 

Converting subhalo (galaxy) luminosity to apparent magnitude

In [28]:
%autoreload 2 

In [8]:
import sys
sys.path.append("../")

In [29]:
from calculate_astrophy_quantities import *

# How bright Illustris galaxies need to be for them to be observable

Keck has a limiting magnitude of 24.4 

In [37]:
24.4 - 5. * (np.log10(lum_distance.value * 1e6) - 1.)

-16.555400636223652

This means that the galaxies have to have an apparent magnitude of at most -16.6
$$ m <= -16.6 $$
to be observable

# Field of view (FOV) of different telescopes

assuming z = 0.3

In [27]:
view = {"width": 34., "height": 27}

In [38]:
subaru_field_of_view(Illustris_cosmology, view=view, z=0.3)

array([ 9.08949905,  7.2181316 ])

Most clusters seem to fit comfortably within the FOV of Subaru

In [39]:
view = {"width": 3., "height": 3.}

In [40]:
Hubble_acs_field_of_view(Illustris_cosmology, view=view, z=0.3)

array([ 0.80201462,  0.80201462])

Observers may need to have multiple pointings if they want to do weak / strong lensing with the HST